code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -409,6 +432,10 @@
 					return
 				if(T in items_preserved)
 					return
+				if(istype(src,/obj/item/device/dogborg/sleeper/compactor/analyzer) && istype(T,/obj/item/weapon))
+					var/obj/item/weapon/tech_item = T
```

Comment:
Why `/obj/item/weapon`? Are there not some `/obj/item` that have techlevels?

Reply:
obj/item popped up undefined var error so not as far as I know.

Reply:
That's strange, because origin_tech is defined on the /item level
https://github.com/VOREStation/VOREStation/blob/master/code/game/objects/items.dm#L21

And /weapon adds almost literally nothing, and is mostly a remnant of oldcode
https://github.com/VOREStation/VOREStation/blob/master/code/game/objects/weapons.dm#L1

Reply:
Huh yeah found the real reason for the error. The thing wasn't specified as obj/item before the part.